docs/source/UIDocumentation.md

```diff
@@ -1,160 +1,238 @@
+* ```addError``` - adds error using ```errorText```
+* ```dismissAllErrors``` - dismisses all errors
+* ```copyTrain``` - copies the nets train option for the test option
 * ```trainOnly```
-* ```saveDb```
-* ```loadDb```
-  * id
-* ```infoModal``` - sets info into state, and then open's modal
-* ```toggleSidebar```
-* ```zooModal```
-* ```handleClick``` - handles a click based on an event handles connections and adding layers. 
-  * event
-### ```canvas.js```
-```content.js``` invokes an instance of ```canvas.js``` that contains the following methods and params:
-* ```allowDrop```
-  * event
-* ```clickLayerEvent``` - invoked on a click of a layer, invokes ```setParams.js```
-  * event
-  * layerId
-* ```hoverLayerEvent``` - invoked on a hover
-  * event
-  * layerId
-* ```scrollCanvas```
-* ```clickCanvas```
-  * event
-* ```updateLayerPosition``` - changes layers positon based on event
-  * event
-* ```connectionEvent``` - invoked on connection, checks for cycles and modifies layers.
-  * connInfo
-  * originalEvent
-* ```detachConnectionEvent``` - modifies layers
-  * connInfo
-  * originalEvent
-* ```drop```
-  * event
-```canvas.js``` also contains the code that decides whether a node's line needs to be rerouted if it is cutting through another node. 
-### ```canvas.js```'s placement algorithm
-The method it uses is the following:
-```checkIfCuttingLine``` is passed in a positional block that includes x and y coordinates (it assumes a px is at the end of each x and y) for each endpoint of the line it is checking. 
-Specifically, it is checking if the line formed with the coordinates in the positional block will cut into any other nodes between them.
+* ```saveDb``` - creates RTC hyperlink
+* ```loadDb``` - loads model for RTC by ```id```
+* ```performSharedUpdate``` - updates RTC shared model using sockets
+* ```performSharedAdd``` - adds shared layer in RTC
+* ```performSharedDelete``` - deletes shared layer in RTC
+* ```addSharedComment``` - adds comment shared in RTC
+* ```toggleSidebar``` - toggles the visibilty of the sidebar
+* ```handleClick``` - handles a click based on an event: handles connections and adding layers.
+* ```render``` - renders webpage using previous functions
 
-```checkIfCuttingLine``` creates then creates an equation from the x and y points by calculating the slope and using point slope form. 
+***
 
-After this, it calls ```getBetween``` to get the nodes between the x and y coordinates of the created line. 
+#### ```data.js```
+```data.js``` contains various different variables filled with data used throughout the application, it stores the layers
 
-The ```getBetween``` also serves the purpose of returning which direction in which the majority of the blocks between are. This is purely for performance, otherwise it would be seperated into a seperate function. 
+***
+#### ```error.js```
+```error.js``` contains Error React Component with two methods:
+* ```dismissError```
```

Comment:
No explanation for this method? Having no explanation looks a bit weird while all other methods have them. Either move this method to the end of the list, or provide a brief explanation.